example/csv/generated/raster/orthomosaic_item/orthomosaic_item.json

```diff
@@ -41,8 +41,6 @@
     6951062.348691959
   ],
   "properties": {
-    "eo:snow_cover": 0,
-    "eo:cloud_cover": 0,
```

Comment:
Not sure what code changes caused these two lines to go away but I think they should still be here.

Reply:
it went away because I used .apply() for eo extension but have fixed it now